woocommerce/woocommerce-android
# Remove order details address validation

Summary
==========
Reverts: https://github.com/woocommerce/woocommerce-android/issues/7257

After evaluating the results of the Address Validation analytics, we considered that our hypothesis failed and decided to remove the validation introduced here to avoid an unnecessary increase in the Order Details complexity.

How to Test
==========
1. Just make sure the Order details works as expected when loading the Order data. 

Update release notes:
